spdk.spec

```diff
@@ -141,27 +140,9 @@ make -C doc
 %make_install %{?_smp_mflags} prefix=%{_prefix} libdir=%{_libdir} datadir=%{_datadir}
 
 # Install tools
-mkdir -p %{install_datadir}
-find scripts -type f -regextype egrep -regex '.*(spdkcli|rpc).*[.]py' \
-	-exec cp --parents -t %{install_datadir} {} ";"
+mkdir -p %{install_datadir}/scripts
 
-# env is banned - replace '/usr/bin/env anything' with '/usr/bin/anything'
-find %{install_datadir}/scripts -type f -regextype egrep -regex '.*([.]py|[.]sh)' \
-	-exec sed -i -E '1s@#!/usr/bin/env (.*)@#!/usr/bin/\1@' {} +
```

Comment:
You shouldn't remove this.  We need to comply with packaging guidelines/requirements and this is correcting a violation of one.

Reply:
we aren't using the scripts , I removed from them from the build

Reply:
It would probably be better to comment these lines out then, so that somebody in the future needing to resurrect any, knows that they need to be fixed also.

Reply:
done